packages/docs/content/components/label.mdx

```diff
@@ -42,14 +42,16 @@ The following CSS variables can be overridden to customize Form components:
 
 ### Legend
 
-- Use a single legend for fieldset (this is required). One example of a common use of fieldset and legend is a question with radio button options for answers. The question text and radio buttons are wrapped in a fieldset, with the question itself being inside the legend tag.
+- Use a single legend for fieldset (required). One example of a common use of fieldset and legend is a question with radio button options for answers. The question text and radio buttons are wrapped in a fieldset, with the question itself being inside the legend tag.
 
 ### Content
 
-- Describe what the user needs to do when the field is selected.
-- Use a phrase or complete sentence.
-- Start with an action verb and pair it with a noun to help it be clearer. Example: `Enter your age.`
-- Use the fewest words possible (i.e. don't use "please").
+- Content style for labels varies based on the user experience and its goals.
+  - Form fields can use traditional labels for many items. (Examples: "First name," "Last name," "Address," etc.) Traditional labels don’t need punctuation.
+  - Form labels and decision helpers can use imperatives or questions for user understanding or conversational feel. (Examples: "Tell us why you need a faster decision." "Select a reason for your request." "Do you want to see costs when you compare X?" "Do you have another mailing address?") With imperatives and questions include punctuation.
+- Use a phrase or complete sentence. If using a complete sentence, include punctuation.
+- For imperative labels, start with an action verb and pair it with a noun to help it be clearer. Example: "Enter your age."
+- Use the fewest words possible. Don't use "please" or add unnecessary explanation/context in your label. Instead, use hint text or a help drawer to provide information the user must have to correctly enter their answer. 
```

Comment:
... this is something to consider as a team, but including "help drawer" here as part of label guidance is an interesting choice. 

afaik we don't have examples showing what this pattern would look like, but my gut thinks this could be interpreted as including drawer buttons in labels? Should we expand our examples to show how this guidance looks in action?

Reply:
Sent some follow-up questions to the content team.

Reply:
Followed up and made changes